client/components/repository/common/ActivityDiscussion.vue

```diff
@@ -11,7 +11,7 @@
       @seen="setLastSeenComment"
       @unresolve="updateResolvement"
       v-bind="{ comments, unseenComments, showHeading, user }"
-      scroll-target="commentInput"
+      scroll-target="commentEditor"
```

Comment:
Technically it is no longer an 'Editor'. Consider renaming inputContainer or some similar